http-api/src/test/java/io/avaje/http/api/context/ThreadLocalResolverTest.java

```diff
@@ -0,0 +1,48 @@
+package io.avaje.http.api.context;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
```

Comment:
A test to verify that it all works with a CompletableFuture being the endpoint’s method return type? Seeing as that code in the generated controller sources is different to everything else.